sdk/storage/storage-file-share/test/fileclient.spec.ts

```diff
@@ -734,7 +734,6 @@ describe("FileClient", () => {
       // tslint:disable-next-line:no-empty
     } catch (err) {
       assert.equal(err.name, "AbortError");
-      assert.equal(err.message, "The operation was aborted.", "Unexpected error caught: " + err);
```

Comment:
is this unrelated?

Reply:
It's related. `node-fetch` seems to choose a different abort error message than browser...my commit message from https://github.com/Azure/azure-sdk-for-js/pull/12038/commits/f99cefc8688982d7d8fad3fa8c4af63096a67bb7.  Previously a different AbortError is caught.
```
Now that aborting streaming is working, we are getting AbortError from
node-fetch which use a different message "The user aborted a request."
than the browser fetch API does. so stop verifying error.message.
```

Reply:
thanks for explaining